frontend/src/app/main/data/workspace/variants.cljs

```diff
@@ -124,15 +111,15 @@
             changes (-> (pcb/empty-changes it page-id)
                         (pcb/with-library-data data)
                         (pcb/with-objects objects)
-                        (clv/generate-add-new-property variant-id options))
+                        (clvp/generate-add-new-property variant-id options))
 
             undo-id (js/Symbol)]
         (rx/of
          (dwu/start-undo-transaction undo-id)
          (dch/commit-changes changes)
          (dwu/commit-undo-transaction undo-id))))))
 
-(defn set-variant-id
+(defn- set-variant-id
```

Comment:
I don't know why does this need to be an event, with it's own undo-transaction. ¿Could it simply be a local function in `transform-in-variant`?

Reply:
Ok, we'll solve this when the refactor in  `transform-in-variant` is done.